### ISSUE TITLE
'node-loader' not loading sqlserverv8.node issues (with webpack/electron)

### ISSUE THREAD

**Issue by larsjaas** [TimelordUK/node-sqlserver-v8#208]
Just thought I should share that I spent more than a full workday trying to get msnodesqlv8 to load in an electron project. I got the usual `msnodesqlv8.node is not a proper Win32 application`. The solution I eventually settled on was to toss out `node-loader` together with `@marshallofsound/webpack-asset-relocator-loader` and just add `native-ext-loader` to the project instead. The webpack rule became
```
   {
     test: /\.node$/,
     use: 'native-ext-loader',
   },
```

I did BTW get node-loader to load msnodesqlv8 by downgrading node-loader all the way down to v0.6.0 but then the bundled package was not portable to other computers since full development system file paths ended up in the bundled javascript. I don't know where the marshallofsound-thing came from (I did not start the project) but I suspect there is some tutorial or getting-started utility that just sets it up. It could also be a culprit.

I therefore suggest maybe adding to the documentation that for those who struggle with getting the module loaded, they can try using an alternative to node-loader, like e.g. native-ext-loader instead.

**Comment by TimelordUK** [comment]
is it related to this
https://github.com/TimelordUK/node-sqlserver-v8/issues/198

i.e. the rebuild required to install the correct native module.

i cant speak for webpack but the module does install and work with electron

**Comment by larsjaas** [comment]
Yes. I'm just saying something with my setup made it totally impossible to work, even with running electron-rebuild or downloading and unpacking binaries, given the correct module version. But as soon as I cleaned out node-loader for native-ext-loader, it started working out-of-the-box from pristine checkouts, no electron-rebuild required.

**Comment by larsjaas** [comment]
Just a side-note on the topic of the `@marshallofsound/webpack-asset-relocator-loader` entry - it seems to come from `electron-forge` standard template, and is also given in the example in the Electron Forge docs about native modules, at https://www.electronforge.io/config/plugins/webpack

I'm currently focused on also getting the oracledb driver working in the electron-forge+webpack+typescript electron project I'm working on (there are issues with that native module too with the architecture I'm on), so hopefully I'll make some more headway on this topic soon (figure out a setup that works with both). Anyways, msnodesqlv8 is working fine for me at the moment, and thanks for pushing/releasing the latest Typescript updates recently..
